### ISSUE TITLE
Ksqldb streams throwing socketTimeoutException after upgrading from version 5.5.2 to 6.1.0 

### ISSUE THREAD

**Issue by manpreet1992** [confluentinc/ksql#7286]
**Describe the bug**
After upgrading ksqldb from version 5.5.2 to 6.1.0, we are facing "Error issuing POST to KSQL server. path:/query" error on executing SELECT command on an existing stream.

**To Reproduce**
Steps to reproduce the behavior, include:
1. Install KSQLDB version 5.5.2
2. Create stream s4 as mentioned below.
```
ksql> CREATE STREAM s4 (c1 VARCHAR, c2 INTEGER) WITH (kafka_topic='c4', value_format='json');
 Message
----------------
 Stream created
----------------
```
3. Set 'auto.offset.reset'='earliest';
```
ksql> SET 'auto.offset.reset'='earliest';
Successfully changed local property 'auto.offset.reset' to 'earliest'. Use the UNSET command to revert your change.
```
4. Executed Select command
```
ksql> SELECT * FROM s4 EMIT CHANGES;
+------------------------------+------------------------------+------------------------------+------------------------------+
|ROWTIME                       |ROWKEY                        |C1                            |C2                            |
+------------------------------+------------------------------+------------------------------+------------------------------+
|1616558901293                 |first                         |Hi                            |12                            |
|1616558907540                 |second                        |Hello                         |24                            |
|1616558915572                 |third                         |Bye                           |36                            |
|1616558920402                 |fourth                        |pp                            |48                            |
^CQuery terminated
ksql> SHOW STREAMS;
 Stream Name | Kafka Topic | Format
------------------------------------
 S4          | c4          | JSON
 STREAM2     | test2       | AVRO
------------------------------------

```
**Perform helm upgrade to ksqldb 6.1.0 **
1. Execute ksql shell command
```
$confluent-5.4.1/bin/ksql http://testksql-ckaf-ksql.test.svc.cluster.local:8088
OpenJDK 64-Bit Server VM warning: If the number of processors is expected to increase from one, then you should configure the number of parallel GC threads appropriately using -XX:ParallelGCThreads=N

                  ===========================================
                  =        _  __ _____  ____  _             =
                  =       | |/ // ____|/ __ \| |            =
                  =       | ' /| (___ | |  | | |            =
                  =       |  <  \___ \| |  | | |            =
                  =       | . \ ____) | |__| | |____        =
                  =       |_|\_\_____/ \___\_\______|       =
                  =                                         =
                  =  Streaming SQL Engine for Apache Kafka® =
                  ===========================================

Copyright 2017-2019 Confluent Inc.

CLI v5.4.1, Server v6.1.0 located at http://testksql-ckaf-ksql.test.svc.cluster.local:8088

Having trouble? Type 'help' (case-insensitive) for a rundown of how things work!
```
2. Executed SET 'auto.offset.reset'='earliest'; command
```
ksql> SET 'auto.offset.reset'='earliest';
Successfully changed local property 'auto.offset.reset' to 'earliest'. Use the UNSET command to revert your change.
```
3. Executed SELECT command
```
ksql>  SELECT * FROM s4 EMIT CHANGES;
**Error issuing POST to KSQL server. path:/query
Caused by: java.net.SocketTimeoutException: Read timed out
Caused by: Read timed out**
```

**Additional context**
After restarting manually, it started working.

**Comment by agavra** [comment]
thanks for reporting this @manpreet1992 - I was not able to reproduce this locally, so if you have any more info please share it. Otherwise, since you were able to be unblocked by restarting manually, I think we'll consider this one lower priority.

**Comment by manpreet1992** [comment]
Sorry @agavra , actually were facing the issue while using the confluent KSQL client of older version (5.4.1). But when we tried it with 6.1.0 KSQL client, it worked fine. You can also try it using with 5.4.1 or can close the issue as its working with 6.1.0 client.

**Comment by agavra** [comment]
Thanks for getting back to us on this! We don't yet support cross-version client/servers, so I'll close this issue out.
